CONTRIBUTING.md

```diff
@@ -67,20 +65,52 @@ All pull requests must pass standard checks:
     * Circleci unit test run (Pytest)
 * Circleci deployment testing (if applicable)
 
-Everything should be runnable in your dev environment, all necessary configuration files are provided
-in VCS. for pylint you may need to manually point it at `pylint.ini`
+Everything should be executable in your development environment, all necessary configuration files are provided
+in VCS. for pylint you may need to manually point at `pylint.ini`
 ## Hygiene
 
 **Please do not squash your commits prior to your PR.**  If this is necessary, it will be brought up during the review process.
 
-Your PR should have sensible commits and messages.
+Your PR should have sensible commits and messages.  Please see the **Pull Requests Naming & Commenting** section below.
 
-All new features are required to contain tests sufficient for newly added code to be evaluated without reducing coverage.
+New features are required to contain tests sufficient for newly added code to be evaluated without reducing coverage.
 
 After branching, you should immediately use `git push -u origin feature/my-awesome-feature` to make that the default upstream ref.
 
 Before opening a PR, rebase onto develop so your PR can be merged fast-forward only, without merge commits.
-The easiest way to do that is to make the main repo a remote using `git remote add upstream git@github.com:fuelrats/pipsqueak` and then running `git pull --rebase upstream develop`.
+The easiest way to do that is to make the main repository a remote using `git remote add upstream git@github.com:fuelrats/pipsqueak` and then running `git pull --rebase upstream develop`.
+
+## Naming & Commenting Standards
+### Pull Requests Naming
+If working from a registered issue, Include the issue name enclosed in brackets in the title of your Pull Request, ie `[SPARK-99] Update to CONTRIBUTING.md`
+
+Otherwise, use the type of request.
```

Comment:
[Suggestion]
```
Otherwise, use the type of request.
```
Perhaps in addition to the issue name?

Reply:
I will fix this to something more descriptive. Perhaps "Otherwise, please use the type of request along with the issue name, such as ``[Doc] Updating CONTRIBUTING.md``"

Reply:
I realise I wasn't clear with my original comment, I mean something along the lines of this, when it is applicable.
```
[Doc] [SPARK-99] Update to CONTRIBUTING.md
```

Reply:
Well, if you're working a 'registered' issue you do have a ticket number.  Otherwise you will not, and have no SPARK identifier to use...unless I still misunderstand?

Reply:
Your original wording had me thinking of either [SPARK-99] _**or**_ [Doc], dependant on whether a Jira issue exists. Whereas, I believe the [Doc] part should exist regardless of a linked Jira issue.

Reply:
Ahh, I see.  I still think this is overkill, because the type of request is also in the branch name.

ie. ``[SPARK-10298301] Update to CONTRIBUTING.md`` is expected to be in a branch with a ``doc/`` prefix, such as ``doc/update_to_contributing``

Reply:
I actually feel like "bugfix/feature/doc" is actually a task better suited for labels, rather than a tag within the title.

Reply:
I agree with @Eearslya here.